assets/js/scores.js

```diff
@@ -1,5 +1,5 @@
 var highScoresList = document.getElementById('highScoresList');
-var highScores = JSON.parse(localStorage.getItem('highScores'));
+var highScores = JSON.parse(localStorage.getItem('highScores')) || [];;
```

Comment:
We should move the contents of this file into script.js... it doesn't need it's own file